cmd/doracled/cmd/root.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	NodePrivKeyFilePath   = "/data/node_priv_key.sealed"
+	OraclePrivKeyFilePath = "/data/.doracle/oracle_priv_key.sealed"
+	OraclePubKeyFilePath  = "/data/.doracle/oracle_pub_key.json"
+)
```

Comment:
I think these consts can be used in many places, not just cmd.
Well, but I'll think a little more about where I can move it.

Reply:
Agreed. How about to make `types` package and move this consts in `consts.go`? or `paths.go`?

Reply:
Yes. That will be fine. Any name seems fine, I like const.go.